shared/api/models/__tests__/DateModel.spec.ts

```diff
@@ -147,41 +155,41 @@ describe('DateModel', () => {
     it('should return exactly count recurrences', () => {
       const recurrenceRule = rrulestr('DTSTART:20230414T050000\nRRULE:FREQ=WEEKLY;BYDAY=MO')
       const date = new DateModel({
-        startDate: DateTime.fromISO('2017-09-27T19:30:00+02:00'),
-        endDate: DateTime.fromISO('2017-09-28T21:30:00+02:00'),
+        startDate: DateTime.fromISO('2017-09-27T19:30:00'),
+        endDate: DateTime.fromISO('2017-09-28T21:30:00'),
```

Comment:
❓ Why did you change this? From the CMS we always get dates with a timezone offset like before 🤔

Reply:
I just tested this without these changes and the tests all succeed. I'd therefore prefer that you revert this change to the DateModel params to correctly reflect the dates we get from the CMS.